### PR TITLE
Mention build heap size in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by emmanuelbernard**
See https://github.com/quarkusio/quarkus/issues/7461

### CODE REVIEW

**Review by gastaldi — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -98,11 +98,13 @@ select the `eclipse.importorder` file as the import order config file.
 
 * Clone the repository: `git clone https://github.com/quarkusio/quarkus.git`
 * Navigate to the directory: `cd quarkus`
+* set Maven heap to 1.5GB `export MAVEN_OPTS="-Xmx1563m"`
```
Comment (geoand) [inline CONTRIBUTING.md]: Maybe it should say `to at least 1.5GB`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -149,6 +151,10 @@ This project is an open source project, please act responsibly, be nice, polite
 
 ## Frequently Asked Questions
 
+* The maven build fails with `OutOfMemoryException`
```
Comment (machi1990) [inline CONTRIBUTING.md]:
```suggestion
* The Maven build fails with `OutOfMemoryException`
```

**Review by machi1990 — COMMENTED** [review]
Added a small suggestion

File: CONTRIBUTING.md (hunk 1 above)
Comment (emmanuelbernard) [reply CONTRIBUTING.md]: I'd prefer we update the number instead of hiding behind a "at least" comment.

Comment (geoand) [reply CONTRIBUTING.md]: Sure, that's fine with me

File: CONTRIBUTING.md (hunk 2 above)
Comment (emmanuelbernard) [reply CONTRIBUTING.md]: Done

**Review by geoand — APPROVED** [review]

**Review by gsmet — APPROVED** [review]

**Comment by emmanuelbernard** [comment]
@gsmet I am a bit lost on the process, should I close the original issue https://github.com/quarkusio/quarkus/issues/7461 ?
If yes, should I assign 1.3.0? I don't see CR1.

**Pull request merged**